components/textfield/index.css

```diff
@@ -12,56 +12,74 @@
-	--spectrum-textfield-border-color-focus-hover: var(--spectrum-gray-900);
-	--spectrum-textfield-border-color-keyboard-focus: var(--spectrum-gray-900);
-
-	--spectrum-textfield-border-width: var(--spectrum-border-width-100);
+	--spectrum-textfield-border-width: var(--spectrum-border-width-200);
+	--spectrum-textfield-corner-radius: var(--spectrum-corner-radius-medium-size-medium);
 
 	/* default spacing */
-	--spectrum-textfield-spacing-inline-quiet: var(--spectrum-field-edge-to-text-quiet);
+	--spectrum-textfield-spacing-inline: var(--spectrum-component-edge-to-text-100);
 	--spectrum-textfield-spacing-block-start: var(--spectrum-component-top-to-text-100);
 	--spectrum-textfield-spacing-block-end: var(--spectrum-component-bottom-to-text-100);
-	--spectrum-textfield-spacing-block-quiet: var(--spectrum-field-edge-to-border-quiet);
 
-	/* default label spacing */
+	/* default field label and help text spacing */
 	--spectrum-textfield-label-spacing-block: var(--spectrum-field-label-to-component);
-
-	/* default help text spacing */
+	--spectrum-textfield-label-spacing-inline-side-label: var(--spectrum-spacing-200);
 	--spectrum-textfield-helptext-spacing-block: var(--spectrum-help-text-to-component);
 
+	/* default icon size */
+	--spectrum-textfield-icon-size-invalid: var(--spectrum-workflow-icon-size-100);
+	--spectrum-textfield-icon-size-valid: var(--spectrum-checkmark-icon-size-100);
+
 	/* default icon spacing - invalid */
-	--spectrum-textfield-icon-spacing-inline-end-quiet-invalid: var(--spectrum-field-edge-to-alert-icon-quiet);
+	--spectrum-textfield-icon-spacing-inline-start-invalid: var(--spectrum-text-to-visual-100);
+	--spectrum-textfield-icon-spacing-inline-end-invalid: var(--spectrum-field-edge-to-alert-icon-medium);
+	--spectrum-textfield-icon-spacing-block-invalid: var(--spectrum-component-top-to-workflow-icon-100);
 
 	/* default icon spacing - valid */
-	--spectrum-textfield-icon-spacing-inline-end-quiet-valid: var(--spectrum-field-edge-to-validation-icon-quiet);
+	--spectrum-textfield-icon-spacing-inline-start-valid: var(--spectrum-text-to-visual-100);
+	--spectrum-textfield-icon-spacing-inline-end-valid: var(--spectrum-field-edge-to-validation-icon-medium);
+	--spectrum-textfield-icon-spacing-block-valid: var(--spectrum-field-top-to-validation-icon-medium);
 
 	/* font styles */
 	--spectrum-textfield-font-family: var(--spectrum-sans-font-family-stack);
 	--spectrum-textfield-font-weight: var(--spectrum-regular-font-weight);
+	--spectrum-textfield-font-style: var(--spectrum-default-font-style);
+	--spectrum-textfield-placeholder-font-size: var(--spectrum-font-size-100);
+	--spectrum-textfield-line-height: var(--spectrum-line-height-100);
+	--spectrum-textfield-line-height-cjk: var(--spectrum-cjk-line-height-100);
+
+	/* set input line-height to the height of the textfield - prevents the cutting off of diacritics in some languages */
+	--spectrum-textfield-input-line-height: var(--spectrum-textfield-height);
```

Comment:
Noting that because of our previous diacritic issues, we use a line height set to the textfield height for the single line (text field) input WITHOUT an option to use a mod, and the `--spectrum-textfield-line-height` token (and mod) are used for the multiline (text area) input.

Reply:
Just to clarify- I am not supposed to see a difference then in the text field, when switching between locales, correct? I don't see that right now (all the line-heights for text field are set to `--spectrum-textfield-height`) and I believe that is expected.

Reply:
Correct, line height should be the same for textfield regardless of locale, but will differ for cjk in the textarea.